docs/nodes/full-node/update-fullnode-with-new-releases.md

```diff
@@ -1,47 +1,72 @@
 ---
-title: "Update Fullnode With New Devnet Releases"
+title: "Update Fullnode With New Releases"
 slug: "update-fullnode-with-new-devnet-releases"
 sidebar_position: 11
 ---
 
 # Update Fullnode With New Releases
 
-When `devnet` is wiped and updated with newer versions, you will need to update your fullnode as well. If you do not, your fullnode will not continue to synchronize with the network. To update your fullnode, follow these steps:
+This document outlines the process for updating your fullnode. For fullnodes running in `devnet`, an additional data wipe step is required as `devnet` is wiped on every release.
 
 ## If you built the fullnode from aptos-core source code
 
-1. Shutdown your fullnode.
+1. Stop your fullnode by running the below command:
 
-2. Delete the data folder (the directory path is what you specified in the configuration file, e.g., `fullnode.yaml`).
+```bash
+cargo stop aptos-node
+```
 
-   - The default data folder is `/opt/aptos/data`.
+2. For users of the Rust binary, pull the latest release appropriate for your network (`devnet`, `testnet`, or `mainnet`):
+
+```bash
+git checkout [network_branch] && git pull
+```
+
+Replace `[network_branch]` with `devnet`, `testnet`, or `mainnet` as applicable, and rebuild the binary.
+
+3. If your fullnode is running in `devnet`, follow the additional steps in the [Additional data wipe steps for `devnet`](#additional-data-wipe-steps-for-devnet) section below.
 
-3. Delete the `genesis.blob` file and `waypoint.txt` file (depending on how you configured it, you might not have this file and may instead have a `waypoint` directly in your configuration file).
+4. Restart your fullnode by running the same start (`run`) command as before:
 
-4. If you use the Rust binary, pull the latest of `devnet` via `git checkout devnet && git pull`, and build the binary again.
+```bash
+cargo run -p aptos-node --release -- -f ./fullnode.yaml
+```
```

Comment:
Personally, I think we should push people rather than to use `cargo run` to start their node instead to build it, then run the binary.  It leaves some separation and lets you rerun without building incase something happens.

```
cargo build -p aptos-node --release
./target/release/aptos-node -f ./fullnode.yaml
```